pkg/stream/http2/stream.go

```diff
@@ -299,6 +299,7 @@ func (conn *serverStreamConnection) handleFrame(ctx context.Context, i interface
 		URL, _ := url.Parse(URI)
 		h2s.Request.URL = URL
 
+		header.Set(protocol.MosnHeaderSchemeKey, scheme)
```

Comment:
Need del `MosnHeaderSchemeKey` header at the `(s *clientStream) AppendHeaders ` function.